### ISSUE TITLE
Agents: when multiple containers are running just using UI command (without specify port) agent logs displays errors related with policies not linked with the agent

### ISSUE THREAD

**Issue by manrodrigues** [ns1labs/orb#711]

**Comment by weyrick** [comment]
Yes this is surely because we run containers with `--host` networking, so the pktvisor instances try to listen on the same port. I see the pktvisor http library will try to use SO_REUSEPORT if it's available, which it is during the docker build. There is probably I way to prevent this when pktvisor builds - let's move this to pktvisor backlog.
